[2021-04-23 21:03:51,420] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T19:00:00+00:00 [queued]>
[2021-04-23 21:03:51,459] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T19:00:00+00:00 [queued]>
[2021-04-23 21:03:51,460] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 21:03:51,460] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-04-23 21:03:51,461] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-23 21:03:51,474] {taskinstance.py:1063} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc> on 2021-04-23T19:00:00+00:00
[2021-04-23 21:03:51,481] {standard_task_runner.py:52} INFO - Started process 674 to run task
[2021-04-23 21:03:51,486] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recent_played_ingestion', 'create_cluster_dataproc', '2021-04-23T19:00:00+00:00', '--job-id', '45', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/recent_played_ingestion.py', '--cfg-path', '/tmp/tmpy8f0w7km', '--error-file', '/tmp/tmpj7x2buss']
[2021-04-23 21:03:51,489] {standard_task_runner.py:77} INFO - Job 45: Subtask create_cluster_dataproc
[2021-04-23 21:03:51,585] {logging_mixin.py:104} INFO - Running <TaskInstance: recent_played_ingestion.create_cluster_dataproc 2021-04-23T19:00:00+00:00 [running]> on host 760ffd3d790b
[2021-04-23 21:03:51,675] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recent_played_ingestion
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc
AIRFLOW_CTX_EXECUTION_DATE=2021-04-23T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-23T19:00:00+00:00
[2021-04-23 21:03:51,680] {dataproc.py:603} INFO - Creating cluster: ephemeral-recent-played-spark-cluster-20210423
[2021-04-23 21:07:59,285] {dataproc.py:547} INFO - Cluster created.
[2021-04-23 21:08:00,527] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=recent_played_ingestion, task_id=create_cluster_dataproc, execution_date=20210423T190000, start_date=20210423T210351, end_date=20210423T210800
[2021-04-23 21:08:01,650] {local_task_job.py:146} INFO - Task exited with return code 1
